app/scripts/views/sign_in.js

```diff
@@ -37,6 +37,28 @@ define(function (require, exports, module) {
 
     beforeRender: function () {
       this._account = this._suggestedAccount();
+
+      // Extract email to see if valid account
+      var relierEmail = this.relier.get('email');
+      var formEmail = this._formPrefill.get('email');
+
+      // Use the relier's email to check for valid account.
+      // If the form email has been set, this signals that the user has modified the form email input
+      // and we want to use it for later updates
+      if (relierEmail && ! formEmail) {
+        var self = this;
+        this.fxaClient.checkAccountByEmailExists(relierEmail)
```

Comment:
Creating an additional call in accounts [also allows you to remove the newly created references to fxaClient to the test code](https://github.com/mozilla/fxa-content-server/pull/3537/files#diff-5691ba30bfcf6bbde5e4d769cb966aceR14).